lib/config/store.go

```diff
@@ -226,6 +226,11 @@ type Store struct {
 	MinChunkReaderConcurrency    int           `toml:"min-chunk-reader-concurrency"`
 	MinShardsConcurrency         int           `toml:"min-shards-concurrency"`
 	MaxDownSampleTaskConcurrency int           `toml:"max-downsample-task-concurrency"`
+
+	// config for lazy load shard
+	LazyLoadShardEnable       bool          `toml:"lazy-load-shard-enable"`
```

Comment:
In fact, there is no such thing as cold shard here, and it is not recommended to add this concept, as it may be confused with cold, hot, and warm shards.